### ISSUE TITLE
Screenshots feature request: add simple edit options

### ISSUE THREAD

**Issue by mailme-gx** [mozilla-services/screenshots#3788]
I would love it if there were simple edit options (like old school paint) to block out sensitive areas, circle or frame an area and add pointers, this means I dont have to download the image and open another editor, it makes for a much easier workflow

**Comment by mailme-gx** [comment]
![simple-editor-example-2017-11-16-05-19-07](https://user-images.githubusercontent.com/4591900/32872135-cb50660a-ca8d-11e7-9e76-3f55acf2c633.png)
Example of a simple editor

**Comment by chenba** [comment]
Hi @mailme-gx, there are plans for "[Annotations](https://github.com/mozilla-services/screenshots/issues?utf8=%E2%9C%93&q=is%3Aissue%20Annotations)", which is similar to what you're describing.  

However, for the purpose of "block out sensitive areas", that should be done before upload, which is not how the Annotations feature currently works.

**Comment by mailme-gx** [comment]
chenba thanks for the clarifications

**Comment by johngruen** [comment]
We have a ticket for this, closing in favor of #1849
